### PR TITLE
media issue without fonts

### PR DESCRIPTION
**Pull request by umizoomiexe**
issue_media

here is how it looks. I did not know what fonts were used in the example phot... still havent found it. please lmk what you think !

### CODE REVIEW

File: src/components/Media.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "next/image";
+import image1 from "@/public/assets/logo.svg";
+const Media = () => {
+  return (
+    <div className="w-full h-full flex-center">
```
Comment (SafeDuck) [inline src/components/Media.jsx:6]: it's using font saira

File: src/components/Media.jsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Image from "next/image";
+import image1 from "@/public/assets/logo.svg";
```
Comment (SafeDuck) [inline src/components/Media.jsx:3]: i want you to use the placeholder image for now

**Review by SafeDuck — CHANGES_REQUESTED** [review]

File: src/components/Media.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const Media = () => {
+  return (
+    <div className="w-full h-full flex-center">
+      <div className="flex justify-center bg-blue-950 px-2 py-2">
```
[the remark this answers is not in the document]
Comment (umizoomiexe) [reply src/components/Media.jsx:7]: the example had a dark blue background?

[the remark this answers is not in the document]
Comment (SafeDuck) [reply src/components/Media.jsx:7]: Keep it without color for now

[the remark this answers is not in the document]
Comment (umizoomiexe) [reply src/components/Media.jsx:7]: do we still want text to be white then... or black

**Pull request closed without merging**